frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -61,7 +61,6 @@ export type MapProps = {
   zoom?: number;
   properties: IProperty[];
   agencies: ILookupCode[];
-  propertyClassifications: ILookupCode[];
```

Comment:
No longer required because they are retrieved via the `useLookupCodes()` hook.  Ideally I'd replace the others too, but keeping this PR as small as possible.